app/components/support_interface/provider_courses_table_component.rb

```diff
@@ -34,15 +34,16 @@ def accredited_bodies_vary?
     attr_reader :provider, :courses
 
     def status_tag(course)
-      # Special case when the course is in the future but has been imported with a status of open_on_apply
-      return govuk_tag(text: 'Closed on Apply', colour: 'blue') if course.recruitment_cycle_year == RecruitmentCycle.next_year
-
-      if course.open_on_apply?
-        govuk_tag(text: open_on_apply_message(course), colour: 'green')
-      elsif course.exposed_in_find?
-        govuk_tag(text: 'Closed on Apply', colour: 'blue')
-      else
+      if course.recruitment_cycle_year == RecruitmentCycle.next_year
```

Comment:
Can we move this method to the model? `recruitment_cycle_year == RecruitmentCycle.next_year`